src/transformers/models/mt5/modeling_mt5.py

```diff
@@ -1334,7 +1339,6 @@ class MT5Model(MT5PreTrainedModel):
 
     model_type = "mt5"
     config_class = MT5Config
-    _keys_to_ignore_on_load_missing = ["decoder.block.0.layer.1.EncDecAttention.relative_attention_bias.weight"]
```

Comment:
bit surprised by this change, could we leave it as is? (for BC)

Reply:
I removed this because of a test failure in test_model_weights_reload_no_missing_tied_weights():
https://github.com/huggingface/transformers/blob/415e9a0980b00ef230d850bff7ecf0021c52640d/tests/test_modeling_common.py#L1956

The reason it is failing is that "decoder.block.0.layer.1.EncDecAttention.relative_attention_bias.weight" is a key that will never exist in the model (or the current version of the code), and that test fails because of that. It's expecting the missing key to be in the reloaded model. 

I believe we should be OK with backwards compatibility because the current version of the code is hard-coded not to have that key:
https://github.com/huggingface/transformers/blob/de13a951b38b85195984164819f1ab05fe508677/src/transformers/models/mt5/modeling_mt5.py#L481

WDYT? The other option would be to find a way to disable that test for the MT5model.